config/templates/operator.yaml.in

```diff
@@ -0,0 +1,316 @@
+  name: cockroach-operator-sa
+  namespace: default
+  annotations:
+  labels:
+    app: cockroach-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cockroach-operator-default
+  labels:
+    app: cockroach-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: cockroach-operator-role
+subjects:
+  - name: cockroach-operator-sa
+    namespace: default
+    kind: ServiceAccount
+
+# Operator Deployment Definition:
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cockroach-operator
+  namespace: default
+  labels:
+    app: cockroach-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cockroach-operator
+  template:
+    metadata:
+      labels:
+        app: cockroach-operator
+    spec:
+      serviceAccountName: cockroach-operator-sa
+      containers:
+        - name: cockroach-operator
+          image: cockroachdb/cockroach-operator:{{ .OperatorVersion }}
+          imagePullPolicy: IfNotPresent
+          # uncomment this to enable different new features
+          # args: ["-feature-gates","PartitionedUpdate=true,UseDecommission=true,Upgrade=false, CrdbVersionValidator=false"]
+          # this is for openshift only
+          # args: ["-feature-gates", "Upgrade=false,ResizePVC=false"]
+
+          # different new features are commented out below
+          # uncomment them to enable the featureGate
+          args:
+            - feature-gates
+            - Upgrade=false
+            # the below log level accepts "info" "debug" "warn" or "error"
+            - -zap-log-level
+            - info
+              # - debug
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: cockroachdb
+{{- range .CrdbVersions }}
+            {{- /*
+                .CrdbVersions: a slice of CockroachDB versions. Every item uses
+                semver.Version as its type. See https://github.com/Masterminds/semver for
+                available methods. We use the `Original()` method, which preserves the
+                leading "v" in the output.
+                The `stable` template function is used to determine if the
+                version is stable or alpha/beta/rc.
+                The `underscore` template function replaces all dots with underscores.
+            */}}
+            - name: RELATED_IMAGE_COCKROACH_{{ underscore .Original}}
+              {{- /*
+                Stable CockroachDB docker images published in
+                cockroachdb/cockroach, and unstable versions go to
+                cockroachdb/cockroach-unstable.
+              */}}
+              value: cockroachdb/cockroach{{ if not (stable .) }}-unstable{{end}}:{{ .Original }}
```

Comment:
/nit this is being a bit too terse 
```
{{ if (stable .) }}
value: cockroachdb/cockroach:{{ .Original }}
{{ else }}
value: cockroachdb/cockroach-unstable:{{ .Original }}
{{ end }}
```
Is more readable to me but this is fine

Reply:
YES! This is much better.